### PR TITLE
Add blog post for November 2015 Extravaganza.

### PR DESCRIPTION
**Pull request by Osmose**
@jgmize r?

I'll be posting this on Wednesday morning, so please leave any feedback or corrections before then. Thanks!

### CODE REVIEW

File: blog/extravaganza/2015-11-November.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Once a month, web developers from across Mozilla get together to revolutionize
+layoffs. While we calculate the optimum distance away an employee should be from
+their desk when they're informed of their impending doom, we find time to talk
```
Comment (jgmize) [inline blog/extravaganza/2015-11-November.md]: While I appreciate the dark humor, there might be some readers who are either have legitimate anxieties of being laid off from their current employer and see this as tone deaf, and others who don't even realize you're joking and think that Mozilla is actually laying people off. Might want to go with something else for the intro.

File: blog/extravaganza/2015-11-November.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Last was Osmose again with the news that [django-browserid 1.0.1][] has been
+released. The only changes are the addition of a universal wheel distribution
+and preliminary Django 1.9 support.
+
+[django-browserid 1.0.1]: https://github.com/mozilla/django-browserid
+
+---
+
+This month's excellence award went to Mythmon's "Severance Santa" plan for
+holding layoffs over the holiday break.
```
Comment (jgmize) [inline blog/extravaganza/2015-11-November.md]: If you decide to take my advice about the intro, you'll probably want to update this as well.

**Comment by jgmize** [comment]
r+wc

File: blog/extravaganza/2015-11-November.md (hunk 1 above)
Comment (Osmose) [inline blog/extravaganza/2015-11-November.md]: I'm not terribly concerned about people seeing it as tone deaf (it's always a little intentionally tone deaf), and I'm certain no one who matters will think Mozilla is actually holding layoffs.

Of course, if I'm wrong, I'll be the one responsible, and am ready to die for my right to mockery.

**Pull request merged**